File: MODUL 1 PEMROGRAMAN 1/PRAK110-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>
#include <math.h>

int main (){
    int alas, tinggi, simi;
    alas = 5;
    tinggi = 12;
    //Mencari sisi miring menggunakan rumus pythagoras
    simi = sqrt(pow(alas, 2) + pow(tinggi, 2));
    printf("Diketahui :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);
    printf("\n\n");
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", tinggi);
    printf("Sisi B = %d cm\n", simi);
    printf("Sisi C = %d cm\n", alas);
    printf("Keliling = %d cm\n",tinggi + alas + simi);
    printf("Luas = %d cm\n", alas * tinggi / 2);
    return 0;
}

File: MODUL 1 PEMROGRAMAN 1/PRAK110-2310817120002-SITI RATNA DWINTA SARI.py
import math

alas = 5
tinggi = 12
#Mencari sisi miring menggunakan Rumus Pythagoras
simi = round (math.sqrt(alas**2 + tinggi**2))
#Keliling = jumalah dari semua sisi
kl = alas + tinggi + simi
#Luas = alas dikali tinggi dibagi 2
luas = round (alas * tinggi / 2)
print("Diketahui :")
print("Alas :", alas, "cm")
print("Tinggi :", tinggi, "cm")
print("\n" * 2)
print("Jawab :")
print("Sisi A =", tinggi, "cm")
print("Sisi B =", simi, "cm")
print("Sisi C =", alas, "cm")
print("Keliling =", kl, "cm")
print("Luas =", luas, "cm")